lib/vdc/BOVCollection.cpp

```diff
@@ -247,6 +251,9 @@ int BOVCollection::_validateParsedValues()
     if (_tmpBrickSize != _brickSize && _brickSizeAssigned == true)
         return _inconsistentValueError(BRICK_SIZE_TOKEN);
     else {
+        for (size_t i = 0; i < _tmpBrickSize.size(); i++) {
+            if (_tmpBrickSize[i] < 0) return _invalidValueError(BRICK_SIZE_TOKEN);
```

Comment:
Q1: is brick size equal to 0 OK?

Q2: You're comparing a double with an integer. What's the expected behavior: is it to cast everything to double and then perform the comparison, or to cast everything to integer and then perform the comparison?

Reply:
1) Yes
2) No problem, I ditched the implicit cast in the current HEAD of this branch.